### PR TITLE
Milestone 2 interactive list

### PR DESCRIPTION
**Pull request by RadiactiveJesus**
**Interactive List**

1. Add a new JavaScript file and import it as a module: it will contain methods related to the status updates (completed: true/false).
2. Add event listener to the checkbox (change).
3. Update items object's value for completed key upon user actions.
4. Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/todoList.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export default class ToDoList {
+  static list = []
```
Comment (SajjadAhmad14) [inline src/todoList.js]: The task object's values for `completed` should be updated upon user actions. The LIst is empty so please add some items and make sure their completed value becomes true when checked and vice versa.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Rodolfo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/todoList.js (hunk 1 above)
Comment (RadiactiveJesus) [reply src/todoList.js]: The list is not empty, if I populate this array my elements will be doubled. Also my tasks are already being kept in track of his completion

Comment (SajjadAhmad14) [reply src/todoList.js]: Please watch this video for clarity https://www.loom.com/share/f56a54c5cd604f25b8aad4211f2a3934

Comment (carlos-ssh) [reply src/todoList.js]: Kindly populate your empty array with an array of objects that should contain: 
- Methods related to the status updates (completed: true / false).
- It has event listener on the checkbox (change).
- The task object's values for completed is updated upon user actions.
- Update the array of items is stored in local storage, so the user gets the correct list values after the page reloads.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @RadiactiveJesus , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @RadiactiveJesus, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

- Your project is working really nice 😄 👍 Amazing job

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Hello 👋,

This is an additional review as a result of the second opinion request.

**Great work making all the necessary corrections from the previous review.**

Looks like the issue was resolved here, so I have nothing else to say other than best of luck with the rest of your projects!

Keep rocking 🚀!

**Comment by RadiactiveJesus** [comment]
Thank you all

**Pull request merged**